test/unit/lib/ToolbarPanelHub.test.js

```diff
@@ -7,7 +7,6 @@ describe("ToolbarPanelHub", () => {
   let sandbox;
   let instance;
   let everyWindowStub;
-  let message;
```

Comment:
Can you please also add a test for checking that the elements are rendered in sorted order?
I would stub `toLocaleDateString` and check it's only called `X` number of times. Or that the content rendered has only `X` `dateEl` elements.